### PR TITLE
added "false" as also not _truthy_

### PR DESCRIPTION
**Pull request by AlanThinks**
This pull request has been automatically created by learn.co.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ parseInt('5', 10) === 5 // true
 
 ## Combining Comparisons
 
-We can combine these comparisons together using `&&` (pronounced "and") and `||` ("or"):
+We can string together these comparisons using `&&` (pronounced "and") and `||` ("or"):
```
Comment (aviflombaum) [inline README.md]: I don't like this new language. Submit the PR without this.

**Comment by aviflombaum** [comment]
Thanks for the commits!!!

File: README.md (hunk 1 above)
Comment (alanthinks) [inline README.md]: This is odd, I didn't add these changes "string together changes".. only the other ones below about adding 'false'

File: README.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ if (something) {
 }
 ```
 
-They work as the name implies: _if_ `something` is _truthy_ (so the boolean `true` or anything other than the empty string (`''`), `0`, `null`, or `undefined`), the code in between the curly braces runs; if not, the code between the curly braces is skipped.
+They work as the name implies: _if_ `something` is _truthy_ (so the boolean `true` or anything other than the empty string (`''`), `0`, `false`, `null`, or `undefined`), the code in between the curly braces runs; if not, the code between the curly braces is skipped.
```
Comment (alanthinks) [inline README.md]: I didn't add those previous "string together" changes.. only the other ones here about adding 'false'... how do I delete that other change from the PR?

File: README.md (hunk 1 above)
Comment (gj) [inline README.md]: Reverted this change.

**Comment by gj** [comment]
Thanks @alanthinks !

**Pull request merged**